plugins/woocommerce/client/admin/client/launch-your-store/hub/main-content/pages/payments-content.tsx

```diff
@@ -87,7 +87,7 @@ const InstallWooPaymentsStep = ( {
 			<div className="launch-your-store-payments-content__step--install-woopayments-logo">
 				<img
 					src={ `${ WC_ASSET_URL }images/woo-logo.svg` }
-					alt="Woo Logo"
+					alt={ __( 'Woo Logo', 'woocommerce' ) }
```

Comment:
Best to use "WooCommerce" instead of "Woo" since "Woo" is the business name and also use a placeholder for it (i.e. "%s logo").

Reply:
It does make sense to use WooCommerce, updated!